css/bootstrap-slider.css

```diff
@@ -243,4 +244,75 @@
   background-repeat: repeat-x;
   filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#ff89cdef', endColorstr='#ff81bfde', GradientType=0);
   opacity: 1;
-}
+  background-color:#89cdef;
+}
+
+.slider.slider-success .in-selection,
+.slider.slider-success .slider-selection.tick-slider-selection,
+.slider.slider-success .slider-handle.min-slider-handle,
+.slider.slider-success .slider-handle.max-slider-handle {
+  background-image: unset;
+  background-color: #62c462;
+  /*background-color: #95d06e;*/
```

Comment:
Those "bootstrap like" css settings where just added as an example.
As you see i am not a huge fan of backgrounds with gradients and therefore i just overruled your settings and unset the background-image. But i think as something like this should be available and at least bootstrap 3 also just uses one simple background-color instead of a gradient those gradient settings and also those bootstrap like css settings should maybe added in an additional files as examples?
This depends on which direction the slider should develop i think and should done by the owner of the extension.